src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/PropagateCancellationTokensWhenPossible.cs

```diff
@@ -33,11 +38,96 @@ public abstract class PropagateCancellationTokensWhenPossibleAnalyzer : Diagnost
 
         public override void Initialize(AnalysisContext analysisContext)
         {
-            // TODO: Enable concurrent execution of analyzer actions.
-            //analysisContext.EnableConcurrentExecution();
+            analysisContext.EnableConcurrentExecution();
 
-            // TODO: Configure generated code analysis.
-            //analysisContext.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.None);
+            // TODO: Include this?
+            // analysisContext.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.None);
```

Comment:
Should this analyzer run on generated code?

Reply:
It should not. Generally, reporting a warning on code the user has no control over is something we try not to do.